Package/Beacon.nuspec

```diff
@@ -16,12 +16,17 @@ To get started just run the following command-line:
 
 `beacon --url=http://yourteamcity.com --username=username --password=password --builds=build_id_1 build_id_2 etc`
 
-    </description> 
+or, alternatively using TeamCity guest access and running only once with verbose logging:
+
+`beacon --url=http://yourteamcity.com --guestaccess --runonce --verbose --builds=build_id_1 build_id_2 etc`
+
+    </description>
     <language>en-US</language>
-    <licenseUrl>https://github.com/dennisdoomen/beacon/blob/develop/LICENSE</licenseUrl>    
+    <licenseUrl>https://github.com/dennisdoomen/beacon/blob/master/LICENSE</licenseUrl>
+    <packageSourceUrl>https://chocolatey.org/packages/beacon</packageSourceUrl>
```

Comment:
This should point to `https://github.com/dennisdoomen/beacon`.

Reply:
Probably also good to add  `bugTrackerUrl` and have it point at `https://github.com/dennisdoomen/beacon/issues`